com.onlyoffice.DesktopEditors.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">com.onlyoffice.DesktopEditors</id>
```

Comment:
AFAIK the current preference is for the above to look more like:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<component type="desktop-application">
  <id>com.onlyoffice.DesktopEditors</id>
  <launchable type="desktop-id">com.onlyoffice.DesktopEditors.desktop</launchable>
```

Reply:
This was based off of Discord's AppStream so I'll rectify that later.